css/styles.css

```diff
@@ -0,0 +1,15 @@
+.hero_img {
+    background-image: url(../img/jumbo-background.jpg);
```

Comment:
its not broken now but ensure you have quotes wrapped around URL, as so: `url('../img/jumbo-background.jpg');`